app/templates/pdf/event_invoice.html

```diff
@@ -17,11 +17,11 @@
 
         body {
           position: relative;
-          margin: 0 auto; 
+          margin: 0 auto;
           color: #001028;
-          background: #FFFFFF; 
-          font-family: Arial, sans-serif; 
-          font-size: 12px; 
+          background: #FFFFFF;
+          font-family: Arial, sans-serif;
+          font-size: 12px;
```

Comment:
Why?

Reply:
Its done during the commit..commit hooks format this file

Reply:
@iamareebjamal